.yarn/versions/6c8bcfdb.yml

```diff
@@ -0,0 +1,35 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/core": patch
+  "@yarnpkg/pnp": patch
```

Comment:
`@yarnpkg/plugin-pnp` needs a release too since the hook got updated. Also, I don't get why the core needs one :thinking:

Reply:
Because otherwise the cli wouldn't be suggested, since it doesn't directly depend on yarnpkg/pnp. Although I guess plugin-pnp would have worked as well.